### PR TITLE
[circle-tensordump] temporary cmake change

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit changes cmake find_package mode temporarily

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/circle-tensordump/CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,12 @@ if(NOT TARGET mio_circle)
   return()
 endif(NOT TARGET mio_circle)
 
-find_package(HDF5 COMPONENTS CXX QUIET)
+find_package(HDF5 COMPONENTS CXX QUIET CONFIG)
+
+if(NOT HDF5_FOUND)
```
Comment (seanshpark) [inline compiler/circle-tensordump/CMakeLists.txt:7]: If you're planning to role back this change later, please add a TODO or NOTE about this change

**Comment by seanshpark** [comment]
title has typo

**Comment by seanshpark** [comment]
what do you mean by `find_package mode temporarily` ? what is `temporarily` ?

**Comment by mhs4670go** [comment]
@seanshpark Before this commit, `find_package` is `MODULE` mode. This PR changes this mode to `CONFIG` mode temporarily.   I used "temporarily" because this find_package is gonna be `nnas_find_package` soon.

**Comment by mhs4670go** [comment]
@seanshpark PTAL:)

**Comment by seanshpark** [comment]
> title has typo

please check the comment's I added

**Pull request merged**